seq/seq.c

```diff
@@ -36,13 +34,13 @@ static double start = 0,
 	     stop = 0,
 	     step = 0;
 
-void main(int argc, char *argv[]);
+void main(int argc, char *const argv[]);
 char *buildfmt(void);
 char *getlgstr(void);
-int afterdecsep(char s[]);
+ssize_t afterdecsep(char s[]);
 void usage(void);
 
-void main(int argc, char *argv[]) {
+void main(int argc, char *const argv[]) {
```

Comment:
Not an useful `const`.

Reply:
Actually, this prevents type mismatching.